### PR TITLE
Linear Search

### PR DESCRIPTION
**Pull request by falguni-k**
I have added the linear search function.

### CODE REVIEW

File: linear_search.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//HEADER FILES
+#include<iostream>
```
Comment (amritansh22) [inline linear_search.cpp:2]: This include needs to removed as iostream is already included in **bits/stdc++.h**

**Review by amritansh22 — CHANGES_REQUESTED** [review]

**Comment by amritansh22** [comment]
Excellent work @falguni-k 
Before merging You have do two things:
<ul>
<li>Make the code changes as suggested above</li>
<li>Star the repository to show your support</li>
</ul>

**Comment by falguni-k** [comment]
Hello, I have made the changes as you have suggested. You may merge now.

**Comment by amritansh22** [comment]
@falguni-k  Everything looks great. Thanks a lot.

**Pull request merged**